test/Infrastructure/LeanCode.Kratos.Tests/KratosWebHookHandlerTests.cs

```diff
@@ -144,20 +154,20 @@ Dictionary<string, StringValues> headers
         return httpContext;
     }
 
-    private sealed class KratosWebHookTestHandler : KratosWebHookHandlerBase
+    public sealed class KratosWebHookTestHandler : KratosWebHookHandlerBase
```

Comment:
It's either public or we add [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")] to project for NSubstitute